public/data/akashtrivedig.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "Akash Trivedi",
+  "type": "personal",
+  "bio": "Full-Stack Web App Developer | Open Source Explorer | DSA - Java",
+  "avatar": "https://github.com/akashtrivedig.png",
+  "links": [
+    {
+      "name": "Check me on GitHub",
+      "url": "https://github.com/akashtrivedig",
+      "icon": "github",
+      "color": "red"
+    },
+    {
+      "name": "Check me on Twitter",
+      "url": "https://twitter.com/akashtrivedip",
+      "icon": "twitter"
+    },
+    {
+      "name": "Check me on Leetcode",
+      "url": "https://leetcode.com/web_spaghetti",
+      "icon": "search"
+    },
+    {
+      "name": "Check me on Codeforces",
+      "url": "https://codeforces.com/profile/akash-trivedi",
+      "icon": "codeforces"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Project : Discord Clone a.k.a ShareSpace",
+      "date": "July 2022",
+      "icon": "discord",
+      "color": "#404eed",
+      "description": "A place to join/create communities of your Interests. Create Groups and if want to discuss something in private then go for DM's.",
+      "url": "https://github.com/akashtrivedig/discord-clone/"
+    },
+    {
+      "title": "Project : Realtime Chat App",
+      "date": "June 2022",
+      "icon": "send",
+      "color": "#3BA55D",
+      "description": "Connecting people online. Direct messages via internet. Supports group creation. Video calling(under development) & Voice calling(under development).",
+      "url": "https://github.com/akashtrivedig/chat-app/"
```

Comment:
There is another broken URL, please update it with the correct one!! 🛠

<details>
<summary> Supporting Image </summary>
<img src="https://user-images.githubusercontent.com/88786642/189286791-8ae5fffd-acd8-4400-9194-1acfed59916a.png" />
</details>

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable